### PR TITLE
test suite integration via Github Actions/Checks

### PR DESCRIPTION
**Pull request by fmigneault**
Everything is running just like TravisCI did, but using Github Actions. 

Only the red `x` remains in commits due to missing display functionality of `allowed-failure` as described in 
https://github.com/actions/toolkit/issues/399

@dbyrns @MatProv @ChaamC @f-PLT
Tagging all of you just to spread the info about methodology. 
See also CRIM DevOps page: [Transfert d'exécution de tests TravisCI vers Github Actions](https://www.crim.ca/confluence/pages/viewpage.action?pageId=58623328)

### CODE REVIEW

**Review by dbyrns — COMMENTED** [review]
Nice to see it works. Will let other approve the PR.

**Comment by MatProv** [comment]
Seems a good lightweight alternative to Travis, interesting result.
Let's hope that GH Actions usage won't be limited..

**Review by MatProv — APPROVED** [review]

**Review by ChaamC — COMMENTED** [review]
Everything looks good.
Good job!

**Review by ChaamC — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            test-case: test-docker
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Intall Dependencies
```
Comment (ChaamC) [inline .github/workflows/tests.yml:93]: Just a small typo here for "Install"

**Review by f-PLT — APPROVED** [review]

**Pull request merged**